tests/phpunit/tests/media.php

```diff
@@ -3625,6 +3630,141 @@ function() {
 		$this->assertSame( 3, $omit_threshold );
 	}
 
+	/**
+	 * @ticket 56930
```

Comment:
```suggestion
	 * @ticket 56930
	 * @covers ::wp_get_loading_attr_default
```

Reply:
Thanks, added in https://github.com/WordPress/wordpress-develop/pull/3560/commits/061565acff172dbc7cf26e354c2827bf706a1d49!